Tests/Engine/ModuleHelp.Tests.ps1

```diff
@@ -234,6 +234,25 @@ Describe 'Cmdlet parameter help' {
 			# To avoid calling Trim method on a null object.
 			$helpType = if ($parameterHelp.parameterValue) { $parameterHelp.parameterValue.Trim() }
 			$helpType | Should -Be $codeType -Because "help for $commandName has correct parameter type for $parameterName"
+
+			foreach($parameterAttribute in $parameter.Attributes) {
+				if ($parameterAttribute.ValueFromPipeline -eq $null) { continue }
+
+				$parameterHelpPipelineInput = if ($parameterHelp.pipelineInput -eq 'True (ByPropertyName, ByValue)') {
```

Comment:
is the string format of this going to be consistent

Reply:
yes and if not the test will fail anyway :-)